src/functions/indBallRank.jl

```diff
@@ -41,7 +41,11 @@ function prox!{T <: Real}(f::IndBallRank, x::AbstractArray{T,2}, y::AbstractArra
     for j = 1:size(x,2)
       y[i,j] = 0.0
       for k = 1:f.r
-        y[i,j] += svdobj[:U][i,k]*svdobj[:S][k]*svdobj[:Vt][j,k]
+          if VERSION > v"0.5"
```

Comment:
check out the merge commit https://github.com/JuliaLang/julia/commit/dd6ceffd4efdc8254387d5061f0ee188fcb44dc9 and run the `contrib/commit-name.sh` script